io/blob.go

```diff
@@ -106,6 +106,10 @@ func (bfs *blobFileIO) Remove(name string) error {
 	return bfs.Bucket.Delete(context.Background(), name)
 }
 
+func (bfs *blobFileIO) Create(name string) (FileWriter, error) {
+	return bfs.NewWriter(name, true, nil)
+}
+
```

Comment:
should we add a `WriteFile` here too?

Reply:
should this be changed to `WriteFile`? 
https://github.com/apache/iceberg-go/blob/3943d7e4cc9fd2e95d44d146a31752554a922705/io/blob.go#L160

Reply:
no. There's a difference:

`WriteFile` is on the IO object directly, based on https://pkg.go.dev/os#WriteFile which creates a file and writes the provided content to it directly. It's essentially the equivalent of

```go
f, err := os.Create(name)
if err != nil {
    return err
}
if _, err = f.Write(contents); err != nil {
    return err
}
return f.Close()
```

Whereas this `Write` method is for incremental writes to a file, implementing the [`io.Writer`](https://pkg.go.dev/io#Writer) interface. Notice that this method is a function on the `file` object, not the `IO` object.

Reply:
oh i see, learned something new today